kernel/examples/read-table-multi-threaded/README.md

```diff
@@ -18,3 +22,31 @@ We also ensure that _only_ the threads have copies of the `Sender`s used to send
 by closing the copy that the main thread has once all the threads have been created. This means that
 we can simply loop over our `RecordBatch` receiver, because it will return results until the last
 thread has exited (which closes that last sender).
+
+# Examples
+
+All paths must start with `file://` and be absolute.
```

Comment:
This makes me reminisce about all the good times I had dealing with the `file://c/..` notations on windows :heart:.